### ISSUE TITLE
PvP Ranking System

### ISSUE THREAD

**Issue by vicochips** [JoeAlisson/L2jOrg#385]
A PVP ranking system was added.

 - PvP rankings can be viewed from the Ranking System UI.

 - If a Lv. 40 or higher character with at least 2 class transfers obtains PVP points, they are counted in the ranking.

 - During a PvP situation, a Final Hit can used to kill a chaotic character in order to gain 1 point.

 - If the level of the character you killed is within a level range 15 levels lower or higher than your own level, an additional 10 points can be obtained.

 - If the character you killed ranks within the top 100 in the character ranking, an additional 10 points can be obtained.

 - PvP ranking is reset every Monday at midnight.

 - Those who ranked 1st - 3rd place in the previous week's ranking will gain a free location sharing benefit for 1 week.

Added "Ranker's Authority," which can only be accessed by Server Ranking Rank 1 characters.

"Ranker's Authority" will be added to the center of the Ranking UI and can only be used within the affiliated server's peace zone and not the World Server.

Using "Ranker's Authority" requires 20,000,000 Adena.

Using "Ranker's Authority" creates an identical clone of the Rank 1 character of the Server Ranking and the creation location is announced to the entire server through a screen message.

The created clone will be identified with a unique pillar of light effect, and a 4-hour "Ranker's Authority" buff is given to characters nearby.

"Ranker's Authority" has a 24-hour cooldown, and the created clone resides within the server for 12 hours.

The "Ranker's Authority" buff skill effects are as follows.

 - STR +1, INT +1, P. Evasion +20, Additional Recovery +10 upon use of HP Potion

Changes were made so that when a Server Level Ranking Rank 1 character uses "Ranker's Authority", 20,000,000 Adena is required.

Changes were made so the Ranker's Authority NPC gives buffs to nearby characters at 10 second intervals instead of 5 seconds.

Improvements were made to the visual effect of the Ranker's Authority buff skill range to make it easier to see.

Fixed so that the PVP score will not increase if the same character is killed within 5 minutes after killing the character in the PVP situation and chaotic inclination with the last blow.

 - The kill / death count increases regardless of 5-minute time limit, and only the PVP point get 5 minute penalty.

The PVP ranking 1st to 3rd place is given a ranking buff that displays unique effects when killing a PC.

**Comment by LeonanCarvalho** [comment]
I don't think this system can be simply changed, today it's based in Level and EXP, and it's not a resource loaded from the server, but a Game Interface that receives data from the server.

Some UI components could be used to create some kind of ranking using a community board interface, that allows more flexible UI from the server-side.

![image](https://user-images.githubusercontent.com/3593706/91980981-85459680-ecfe-11ea-8678-9d34427afff9.png)

**Comment by vicochips** [comment]
Hello,

Well, it is using the same rankin system currently working on the server, it is the official UI. But now there will be pvp too
